requirements.txt

```diff
@@ -1,5 +1,5 @@
 coloredlogs==15.0.1
-homeassistant==2025.1.0
+homeassistant==2025.5.3
```

Comment:
As indicated by the CI failure, the version also needs to be updated in hacs.json.